.github/workflows/python2lint.yml

```
@@ -14,7 +14,7 @@ jobs:
         uses: fkirc/skip-duplicate-actions@master
         with:
           github_token: ${{ github.token }}
-          paths: '["kolibri/**.py"]'
+          paths: '["kolibri/**/*.py"]'
```

Do we know what process is doing the expansion?

I've run into issues before with older bash versions or `globstar` not being set, either of which would require:

```suggestion
          paths: '["kolibri/*.py", "kolibri/**/*.py"]'
```

ref: https://unix.stackexchange.com/questions/117826/bash-globstar-matching

---

It uses the pattern match described here: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#patterns-to-match-file-paths see the specific example for `docs/**/*.md`, so the syntax I have is what we want.